app/views/pages/_schema_info.html.erb

```diff
@@ -1,205 +1,159 @@
-     </p>
-
-   <h2>Information about dates</h2>
-   <p>
-   When clinicaltrials.gov started tracking studies, it only tracked the month & year for some fields and not the day for several date values including start date, completion date, primary completion date and verification date. Because day is not provided for some studies, AACT stores these dates in the studies table as character type rather than date type values. Character type dates are of limited utility in an analytic database because they can’t be used to perform standard date calculations such as determining study duration or the average number of months for someone to report results or identify studies registered before/after a certain date.
-   </p>
-
-  <p>We provide 2 columns in the Studies table for each date element:</p>
-
-  <ol>
-    <li>A character-type column that displays the value exactly as it was received from ClinicalTrials.gov</li>
-    <li>A date-type column that can be used for date calculations.</li>
-  </ol>
-
-  <p>
-    If the date received from ClinicalTrials.gov has only month/year, in order to convert the string to a date, it is assigned the first day of the month. For example, a study with start date “June 2014” will have “June 2014” in the start_month_year column and “06/01/14” in the start_date column.
-  </p>
-
-    <div id='pending_results'>
-      <h2>Information about dates related to 'pending results'</h2>
-      <p>On May 9, 2018, the NLM added a new section of date information to the ClinicalTrials.gov API. The section is labeled "pendng_results" and serves to provide information about result submission activity while the results await quality control review.</p>
-
-      <p>NLM provides result submission date(s) for studies that have results awaiting quality control (QC) review. The results themselves are not publicly posted until the review is complete. The dates for three types of events related to results submission are reported in the Pending_Results table:</p>
+    <h4 id='structure_conventions'>Structural Conventions</h4>
+    <ul class='regularDisplay'>
+      <li>All tables include an <i>nct_id</i> column linking rows to the related study in the <i>Studies</i> table. (The schema diagram omits some lines to reduce complexity.)<br>
+        Example: <i>Studies.nct_id = Outcomes.nct_id</i> links outcomes to their study.</li>
```

Comment:
it doens't look like this is true anymore, we are including all the lines in the diagram.